tests/crypto_test.py

```diff
@@ -234,7 +235,7 @@ def _get_key_location(self, device):
 
         return m.group(1)
 
-    @unittest.skipIf("SKIP_SLOW" in os.environ, "skipping slow tests")
+    @tag_test(TestTags.SLOW)
```

Comment:
What is the key to say if the test should be `CORE` or not? It should also be explained in the commit message.

Reply:
That's a good question. I think it's not possible to create some rules for this. I basically went through all the tests and if a test looked "important" I marked it as "core".

Reply:
I'm okay with that, but I still think it should be mentioned in the commit message.

Reply:
OK, I've added detailed description of every tag to the commit message.